benchmarks/docker/test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# This script runs the dockerized benchmarking framework, which in
+# turn will pull Nighthawk and Envoy in via docker. 
+
+set -eo pipefail
+set +x
+set -u
+
+# The benchmark logs and artifacts will be dropped here
+OUTDIR="/home/oschaaf/code/envoy-perf-vscode/nighthawk/benchmarks/tmp/"
+# Used to map the test that we want to see executed into the docker container
+TEST_DIR="/home/oschaaf/code/envoy-perf-vscode/nighthawk/benchmarks/test/"
+
+# Rebuild the docker in case something changed.
+./docker_build.sh && 
+docker run -it --rm \
+  -v "/var/run/docker.sock:/var/run/docker.sock:rw" \
+  -v "${OUTDIR}:${OUTDIR}:rw" \
+  -v "${TEST_DIR}:/usr/local/bin/benchmarks/benchmarks.runfiles/nighthawk/benchmarks/external_tests/" \
+  --network=host \
+  --env NH_DOCKER_IMAGE="envoyproxy/nighthawk-dev:latest" \
+  --env ENVOY_DOCKER_IMAGE_TO_TEST="envoyproxy/envoy-dev:f61b096f6a2dd3a9c74b9a9369a6ea398dbe1f0f" \
```

Comment:
When and who will update this SHA?

Reply:
This is actually an accidental test script that got committed. Maybe it can serve a purpose though, as a NH CI task which ought to succeed. The Envoy sha could then be sourced from our dependency or some such. Regardless it should at least be a variable, maybe defaulting to `latest`

Reply:
I think drop it, but you can use fragments in a doc perhaps.

Reply:
I'll move this into README.md. In the future, we could move this and turn it into a ci task: we fetch a simple 101 test via http, and check that it works like we expect, to at least have some check in place this doesn't break.